__tests__/components/ExportURLInput.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { mantineWrap } from '../helpers/testHelpers';
+import ExportURLInput from '../../components/ExportURLInput';
+
+describe('ExportURLInput', () => {
+  it('renders a text input field', () => {
+    render(mantineWrap(<ExportURLInput />));
+
+    const textbox = screen.getByPlaceholderText('Export URL (Data Source)');
+    expect(textbox).toBeInTheDocument();
```

Comment:
We could also use [fhirEvent](https://testing-library.com/docs/example-input-event/) to simulate an `onChange` event and make sure that the text in the textbox updates